README.md

```diff
@@ -48,18 +52,26 @@ $ $GOPATH/bin/terraform-provider-sysdig
 ...
 ```
 
+### Tests
 In order to **test** the provider, you can simply run `make test`.
 
 ```sh
 $ make test
 ```
 
+### Acceptance Tests
+
 If you want to execute the **acceptance tests**, you can run `make testacc`.
+- Follow [Terraform acceptance test guideliness](https://www.terraform.io/plugin/sdkv2/testing/acceptance-tests)
+- Please note that you need a token for Monitor and Secure, and since the **acceptance tests create real infrastructure**
```

Comment:
Maybe we could explicitly that we need a token for Secure and another one for Monitor.